modules/ROOT/pages/configure-multiple-credential-providers.adoc

```diff
@@ -0,0 +1,80 @@
+
+
+== The Use Case
+
+To better understand how to implement the multiple external credential provider feature, let us go back to our example of "Good Weather" company that is in the business of forecasting weather for a given location and time. Good Weather collects quantitative data about the current state of the atmosphere at a given place and, using meteorology, predicts how that current state will change over time.
+
+Good Weather has several sub-organizations running under the master organization. These sub-organizations exist based on the functions they perform, for example, finance, HR, and R&D. The R&D department researches and collects recent meterological data from various sources. This data is then consumed by various client applications. 
+
+Therefore, Good Weather needs to ensure that this data is accurate at all times. For that reason the weather APIs must be accessed only by employees of the R&D department. Good Weather also want to ensure that the production APIs and the single source of truth at all times.
+
+On the other hand, the HR sub-organization wants to ensure that the employees in R&D do not have access to their payroll APIs. HR also wants to restrict access to their internal tax and payroll APIs.
+
+To ensure security and segregation of access to these different organizations, environments, and APIs, Good Weather IT sets up a separate external IdP for each sub-organization and environments (production and non-production).
+
+
+== Guidelines for Implementing Multiple Client Providers
+
+Before you implement multiple client providers in your instance of Anypoint Platform, ensure that you carefully review the following guidelines and best practices:
+
+* If you assign an external IdP to an environment, all existing APIs in that environment display Anypoint Platform in the UI dropdown. 
++
+You can then reassign the required external IdP for that API.
+* After you assign a external IdP to an API, you cannot change this IdP back to be the default of Anypoint Platform as a credential provider.
+* If you remove an IdP from an environment, all existing APIs and client applications using that IdP will continue working.
+* If you delete an IdP from a master organization, all existing APIs and client applications using that IdP will change to the default Anypoint Platform IdP . 
++
+All contracts will remain intact.
+* When changing the IdP for an existing API instance, ensure that:
+** The API has no contracts
+** The API has no IdP-related policies applied, such as PingFederate and OpenID policies.
+* When promoting and importing an API:
+//[PA: Tomas what are the rules? I was not able to access the LucidChart]
+
+
+== Configuring External Client Providers User Flow
+
+Based on the use case explained in the previous section, Good Weather uses the following flow to implement multiple external IdPs:
+
+image::multiple-IdP-user-flow.png[User Flow for Configuring Multiple External Client Providers]
```

Comment:
the second and third node in the image duplicate?

Reply:
Done! Fixed!